indico_zodbimport/modules/event_regforms.py

```diff
@@ -965,6 +970,9 @@ def load_data(self):
         self.print_step("Loading some data")
         self.default_currency = payment_settings.get('currency')
         self.all_users_by_email = {}
+        self.participant_list_disabled = {x for x, in db.session.query(MenuEntry.event_id)
```

Comment:
I'd put it at the end of the function. Right now it's in the middle of the all_users related code